training/distributed_training/pytorch/data_parallel/maskrcnn/pytorch_smdataparallel_maskrcnn_demo.ipynb

```diff
@@ -304,7 +402,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.4"
+   "version": "3.6.13"
```

Comment:
Why. change the python version ?

Reply:
That wasn't intentional, I believe that was an artifact of using the conda_pytorch_latest_p36 kernel. That's not explicitly necessary.